_sections/home-english.md

```diff
@@ -140,7 +140,7 @@ XLSForm supports both **select_one** (select only one answer) and **select_multi
 
 Note that the **yes_no** in the **survey** worksheet must match the **yes_no** in the **list name** column in the **choices** worksheet. This ensures that the form displays the correct list of answer choices for a particular question.
 
-We can also add multiple choice questions that allow multiple answers to be selected, like so:
+We can also add multiple choice questions that allow multiple answers to be selected.  The responses for select multiple question will be the space-separated choices made by the user, in the order that they were selected:
```

Comment:
The order is not defined by the spec. Collect saves choices in the order they're selected but Enketo saves them in the original order.